modules/ovn-kubernetes-logical-architecture-con.adoc

```diff
@@ -0,0 +1,68 @@
+:_content-type: CONCEPT
+[id="ovn-kubernetes-logical-architecture-con_{context}"]
+= OVN-Kubernetes logical architecture
```

Comment:
We chose not to include this diagram: https://docs.google.com/drawings/d/1cdjt_fU3DxYbv1XExW9IKG-G3KgoifdFv1WytSJVHJ8/edit?usp=sharing.  Too bad, I think it really fits well in this section because it has labels on the types of switches, routers and ports.  Could we perhaps get this in the graphics design pipeline and publish without it and then get it in the doc when it's complete?

Reply:
@CFields651 I think we agreed on the first two but I think we can address this in a follow up PR. I will get this into the graphics design pipeline